packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -29,66 +29,67 @@
 		"min": "3.5",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"paramInformation": {
 		"1": {
-			"label": "Minimum brightness level",
-			"description": "set automatically during the calibration",
+			"label": "Minimum Brightness Level",
+			"description": "For manual override, this is set during auto-calibration",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 98,
 			"defaultValue": 1
 		},
 		"2": {
-			"label": "Maximum brightness level",
-			"description": "parameter is set automatically during the calibration process",
+			"label": "Maximum Brightness Level",
+			"description": "For manual override, this is set during auto-calibration",
 			"valueSize": 1,
-			"minValue": 1,
+			"minValue": 2,
 			"maxValue": 99,
 			"defaultValue": 99
 		},
 		"3": {
-			"label": "Incandescence level of dimmable CFL*",
+			"label": "Incandescence Level of Dimmable CFL*",
 			"description": "Virtual value (percentage between 1% and 99%)",
```

Comment:
Take out the percentage description. @AlCalzone maintains it can't be a percentage if the range is 1-99 and it is his show. Just call it levels. I think you can lose this whole description, though.

Reply:
Would make sense to mention that this is a value relative to the range defined by parameters 1/2.
1 = minimum brightness (param 1), 99 = maximum brightness (param 2)

Reply:
While I fully agree with your point that percentages should go from 0 to 100 and not from 1 to 99 or something else. I don't understand why it's not allowed to call the unit for a parameter as being a percentage point when setting it to 50 means that it is 50% of the maximum value. Just because values 0 and 100 is not allowed for the parameter I don't believe it is wrong to call it a percentage.

Anyway, I changed them all to this:

`Value relative to the range defined by minimum and maximum brightness level configuration`